contracts/interfaces/IZapFi.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I ended up not using this interface but we might want it in the future and its not trivial to translate vyper  code (which this contract is written in) to solidity. Lets keep it, we never know when we might need to use Zap from smart contracts

Reply:
+1

There are Solidity interface implementations for Curve floating around  - like the one our friends in Origin have created: https://github.com/OriginProtocol/origin-dollar/blob/81fe0a94be7825e1023bc31aeaa6d623340de582/contracts/contracts/strategies/ICurvePool.sol

But generally what you did is the exact solution - create the iterface based on the ABI

like: https://chowdera.com/2020/12/20201210232956526b.html